pkg/option/config.go

```diff
@@ -1231,7 +1231,15 @@ func LogRegisteredOptions(entry *logrus.Entry) {
 	}
 	sort.Strings(keys)
 	for _, k := range keys {
-		entry.Infof("  --%s='%s'", k, viper.GetString(k))
+		msgs := viper.GetStringSlice(k)
+		if len(msgs) > 0 {
+			entry.Infof("  --%s='[%s]'", k, strings.Join(msgs, ","))
+			continue
+		}
+		msg := viper.GetString(k)
+		if len(msg) > 0 {
+			entry.Infof("  --%s='%s'", k, msg)
+		}
```

Comment:
I think we don't need to wrap the option list with `[...]` in the output. Also, we should still report options with empty values. How about making this a bit more concise, e.g. as follows?

```suggestion
		v := viper.GetStringSlice(k)
		if len(v) > 0 {
			entry.Infof("  --%s='%s'", k, strings.Join(v, ","))
		} else {
			entry.Infof("  --%s='%s'", k, viper.GetString(k))
		}
```

Reply:
Looks good! I've update the PR, please take a look.